src/rai/rai/tools/ros/manipulation.py

```diff
@@ -0,0 +1,191 @@
+# Copyright (C) 2024 Robotec.AI
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Literal, Type
+
+import numpy as np
+import rclpy
+import rclpy.callback_groups
+import rclpy.executors
+import rclpy.qos
+import rclpy.subscription
+import rclpy.task
+from geometry_msgs.msg import Point, Pose, PoseStamped, Quaternion
+from langchain_core.tools import BaseTool
+from pydantic import BaseModel, Field
+from rai_open_set_vision.tools import GetGrabbingPointTool
```

Comment:
This isn't right in terms of dependencies, as we have a core tool that requires an extension (not optionally)